Tools/TestWebKitAPI/Tests/WebKitCocoa/EventAttribution.mm

```diff
@@ -259,7 +260,7 @@ static void signUnlinkableTokenAndSendSecretToken(TokenSigningParty signingParty
         (__bridge id)kSecAttrKeyType: (__bridge id)kSecAttrKeyTypeRSA,
         (__bridge id)kSecAttrKeyClass: (__bridge id)kSecAttrKeyClassPublic
     }, nil));
-    Vector<uint8_t> rawKeyBytes(static_cast<const uint8_t*>(publicKey.get().bytes), publicKey.get().length);
+    Vector<uint8_t> rawKeyBytes = vectorFromNSData(publicKey.get());
```

Comment:
Should get rid of the local variable and `WTFMove` here since this is so short now.